data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,542 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	"What other short stories have you written?"`
+					goto "short stories"
+				`	"How about your previous books?"`
+			label "previous books"
+			`	"They weren't really successful. I'm still struggling. Some people buy them and I make a little money, but I never really reach the 5-digits. The first one remains unpublished; all the publishers rejected it. The second one was about a kid from the Paradise Worlds being sent to the Dirt Belt by his parents for being spoiled, the third one was about that same kid being kidnapped by pirates. I stopped that series since it was clear it would never sell. And the fourth one was such a failure I don't even want to tell you about it.`
+			`	"'Beyond the Hyperdrive' is the fifth one, and I'm hoping it will sell better than the previous four."`
+			choice
+				`	"How about short stories?"`
+			label "short stories"
+			`	"I write short stories and put them up on the hypernet sometimes. Helps me improve my writing and get that urge satiated without committing to a project as long-term as a novel."`
+			`	Anna shows you a webpage with all her stories. The viewer counts appear to be in the hundreds with the occasional one reaching four digits.`
+			choice
+				`	"So why is 'Beyond the Hyperdrive' about humans stealing from the Quarg?"`
+					goto "quarg"
+			label "book length"
+			`	"About three hundred pages. Not too long."`
+			choice
+				`	"Have you written any other novels?"`
+					goto "other novels"
+				`	"Why is it a story about humanity stealing from the Quarg?"`
+			label "quarg"
+			`	"I have a lot of problems with them. I just don't trust them. I remember when my parents took me to Humanika. Naturally, I wanted to explore, so I wandered off. I was lost. This creepy-looking Quarg found me, and when it tried to approach, I cried.`
+			`	"And then when I grew up, when I was a teenager, I decided to do a project on them for college. They responded with silence to my questions, as if they had something to hide. I worked as an assistant for a journalist for a few months. He kept asking me for help with the Quarg. They just would not budge."`
+			choice
+				`	"That must have been rough, stuck on Humanika. Did your parents eventually find you?"`
+					goto "parents"
+				`	"Well that's all well and good. But I have my own business to attend to now. Good luck with your book, thanks for your time, and goodbye."`
+					decline
+				`	(Tell her about the Coalition.)`
+					goto "coalition"
+			label "parents"
+			`	"They eventually found me, thankfully. But it was distressing being stuck with this Quarg for 10 to 30 minutes, I don't remember how long, before eventually this other Quarg took my parents to me. I told them I wanted to get off Humanika as fast as possible and get back to Zug. They obliged, and we were off that world and I never wanted to come back."`
+			choice
+				`	"So you said your book doesn't sell well, right?"`
```

Comment:
I think one of the following would be clearer, depending on your intended meaning:

```suggestion
				`	"So you said your books don't sell well, right?"`
```

```suggestion
				`	"So you said your book hasn't sold well, right?"`
```

```suggestion
				`	"So you said your book didn't sell well, right?"`
```